magellan-library/src/main/java/com/wealthfront/magellan/navigation/LoggingNavigableListener.kt

```diff
@@ -2,50 +2,37 @@ package com.wealthfront.magellan.navigation
 
 import android.content.Context
 import android.util.Log
-import com.wealthfront.magellan.coroutines.CreatedLifecycleScope
 import com.wealthfront.magellan.init.shouldLogDebugInfo
 import com.wealthfront.magellan.lifecycle.LifecycleAwareComponent
-import com.wealthfront.magellan.lifecycle.lifecycle
-import kotlinx.coroutines.flow.collect
-import kotlinx.coroutines.launch
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
 public class LoggingNavigableListener @Inject constructor(
-  private val navigationTraverser: NavigationTraverser,
-  createdScope: CreatedLifecycleScope
-) : LifecycleAwareComponent() {
-
-  private val createdScope by lifecycle(createdScope)
+  private val navigationTraverser: NavigationTraverser
+) : NavigationListener, LifecycleAwareComponent() {
```

Comment:
Same here, can probably just make this LifecycleAware

Reply:
changed